capitolwords_ng/capitolweb/legislators/urls.py

```diff
@@ -3,6 +3,8 @@
 from . import views
 
 urlpatterns = [
-    url(r'^person/(?P<name>\w+)/$', views.find_by_name, name='search_by_speaker')
-
+    url(r'^person/(?P<name>\w+)/$', views.find_by_name, name='search_by_speaker'),
```

Comment:
If we're using bioguide_id as the main identifier for people, I think it would make more sense for the canonical resource location for a person to be `/legislators/person/<bioguide_id>` and then to present find by name and by id (what is this id -- just a surrogate/autoincrement key?) as search endpoints.

Basically I'm advocating generally (for a give REST `entity_type`):
* `/legislators/entity_type/primaryidentifier`: canonical request for a single entry by primary id -- normally these ids would be primary keys but if we have external ids that we want to think of as the primary way of identifying them, maybe we use those?
* `/legislators/entity_type/search?`: pass query params to search on, e.g. `name=kennedy`; returns a list of matches.

Thoughts? Basically I just think it's weird that the most straightforward resource url here is for a search by name, and not for pulling a specific person by identifier.